### PR TITLE
feat: add topic element activities

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

File: udata/core/topic/apiv2.py [hunk 1]
```diff
@@ -157,6 +154,7 @@ def delete(self, topic):
         if not TopicEditPermission(topic).can():
             apiv2.abort(403, "Forbidden")
 
+        # TODO: should we ignore activity creation here?
```
Comment (abulte) [inline udata/core/topic/apiv2.py]: This will create a _lot_ of deleted activities (and probably crash since it is synchronous) if we empty a universe Topic. I'm not sure we should introduce an exception here to prevent that, WDYT?

Emptying a universe crashes anyway 😬 #3423

File: udata/core/topic/apiv2.py [hunk 2]
```diff
@@ -130,16 +130,13 @@ def post(self, topic):
...
                 element.save()
                 elements.append(element)
 
         if errors:
             apiv2.abort(400, errors=errors)
 
-        for element in elements:
-            element.topic = topic
-            element.save()
-
```
Comment (maudetes) [inline udata/core/topic/apiv2.py:142]: I'm confused, we used to abort if there were errors and save the `topic` field only after.
Seems like we're changing the behavior here but what was the reason before?

File: udata/core/topic/apiv2.py (hunk 1 above)
Comment (maudetes) [reply udata/core/topic/apiv2.py]: Hmm... I think it can be dealt at the same time as https://github.com/opendatateam/udata/pull/3423 indeed?

File: udata/core/topic/apiv2.py (hunk 2 above)
Comment (abulte) [reply udata/core/topic/apiv2.py:142]: I _think_ this is a left over from the first architecture where the TopicElements were embedded in the Topic. I also _think_ I had a test for that and none fail. I don't see any good reason to keep it if the element validation works, as the tests seem to validate.

File: udata/core/topic/apiv2.py (hunk 1 above)
Comment (abulte) [reply udata/core/topic/apiv2.py]: I changed the TODO wording.

File: udata/core/topic/apiv2.py (hunk 2 above)
Comment (maudetes) [reply udata/core/topic/apiv2.py:142]: :crossed_fingers:
